services/core-api/app/api/verifiable_credentials/models/connection.py

```diff
@@ -0,0 +1,26 @@
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from app.api.utils.models_mixins import AuditMixin, Base
+
+
+class MineVerifiableCredentialConnection(AuditMixin, Base):
+    """Verificable Credential reference to Traction, a Multi-tenant Hyperledger Aries Wallet"""
+    __tablename__ = "mine_verifiable_credential_connection"
+    mine_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('mine.mine_guid'), primary_key=True)
+
+    connection_id = db.Column(db.String)
+    connection_state = db.Column(db.String, server_default=FetchedValue()) 
+    #ARIES-RFC 0023 https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange
+
+    invitation_id = db.Column(db.String)
+    
+    def __repr__(self):
+        return '<MineVerifiableCredentialConnection mine_guid=%r, connection_state=%r>' % self.mine_guid, self.connection_state or "UNKNOWN"
+
+    @classmethod
+    def find_by_mine_guid(cls, mine_guid):
+        return cls.query.filter_by(mine_guid=mine_guid, deleted_ind=False).all()
```

Comment:
I'm seeing on a failing test that deleted_ind doesn't exist here, but I'm not sure that the table exists either. I think a migration will be needed to add it. `migrations/sql/V2023.06.29.14.34__add_mine_document_version_table.sql` should be a good example of the format.

Reply:
Fixed that one. I wasn't thinking this needs soft-deletes, but i could implemented it with soft-delete in mind.